.github/workflows/ci.yml

```diff
@@ -71,8 +71,10 @@ jobs:
 
       - name: Run unittest tests with coverage
         run: |
-          pytest --cov=autogpt --without-integration --without-slow-integration --cov-report term-missing --cov-branch --cov-report xml --cov-report term
-
+          coverage run -m pytest --without-integration --without-slow-integration autogpt
```

Comment:
Wait we can’t wait bc this overrides the coverage file. I’d say run them in the same suite for now and with @Pwuts coveeage refactor we can separate it